cmd/org/cmd_test.go

```diff
@@ -0,0 +1,38 @@
+package org
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewCmdOrg(t *testing.T) {
```

Comment:
This test will just be toil to maintain for little to no value, let's remove it.